### DISCUSSION TITLE
CSV header is missing from output

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [github/codeql#10380] [General]
I'm using `--format=csv` to save the output to a CSV file, the file is generated but no header present. 
I'm expecting the first line to be something like `Type,Description,Level,Details,Filename,X,X,X,X`
Please fix this.

**Comment by aibaars** [comment]
Good point, I would have expected there to be a `--[no]-csv-header` flag or similar, but it doesn't seem to exist.

For your use-case would the low-level `codeql bqrs decode` command be useful? That command has `--format=csv` and has `--no-titles`, which suggests it would print column headers by default. You can create a bqrs  (binary query results) file by running `codeql query run`. Alternatively, you can find cached bqrs files by looking in the database folder after running a `codeql database analyze/run-queries` command.

**Reply by CaledoniaProject** [reply]
I want to generate a HTML file. Without vscode the JSON file is very hard to read.
